src/tests/topp/IDFileConverter_9_output.mzid

```diff
@@ -36,22 +36,22 @@
 	</DBSequence>
 	<Peptide id="PEP_14811341817612382122" name="PEPTIDERR">
 		<PeptideSequence>PEPTIDERR</PeptideSequence>
-	</Peptide> 
+	</Peptide>
  	<Peptide id="PEP_15050004366391181853" name="N[3899.3500281914]PEPTIDEK">
 		<PeptideSequence>NPEPTIDEK</PeptideSequence>
 		<Modification location="1" residues="N" monoisotopicMassDelta="3785.3071">
-			<cvParam cvRef="MS" accession="MS:1001460" name="unknown modification" value="N-Glycan"/>
```

Comment:
Yes. Our mzid implementation would add this for every "unknown modification", which doesn't make sense to me. See here: https://github.com/OpenMS/OpenMS/pull/2581/files#diff-2cd91736b9c4a6caf19b1ff80df95be3L924